### PR TITLE
README.md expanded. Updated project information.

### PR DESCRIPTION
**Pull request by klinger**
(just a bit more basic project information)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
 
-A platform to schedule shifts of volunteers. 
+Volunteer Planner is a platform to schedule shifts of volunteers. Volunteers register at the platform and choose shifts.
+ The admin of the website can easily add new organizations, places and shifts. The software has a location based 
+ hierarchy (country / region / area / city) and has a hierarchy of organizations (organization / facility / workplace) 
+ - it can be used for a variety of purposes.
 
-**TODO**: Add general project description and goals, ie. link to wiki.
+## Status
+The software is usable and is used for production use.
```
Comment (christophmeissner) [inline README.md]: > The project is currently running at https://volunteer-planner.org/.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+The software currently needs a decentralized administration of the shifts, it is one of the main goals of the current 
+development to empower organizations to schedule shifts for their facilities on their own.
+
+If you are interested to join the development team, just make pull requests or come to a meeting in Berlin/Germany: 
+http://www.meetup.com/de/coders4help/
+
+## System context
+**User**: The volunteers and administrators just need a (modern) web browser to use the volunteer-planner application.
+
+**Developer**: Developers need a unix-based system, a python development environment (see project setup) and specific
```
Comment (christophmeissner) [inline README.md]: > Developers need ~~a unix-based system,~~ a python development environment

File: README.md [hunk 3]
```diff
@@ -1,10 +1,38 @@
 # volunteer-planner.org
 
-A platform to schedule shifts of volunteers. 
+Volunteer Planner is a platform to schedule shifts of volunteers. Volunteers register at the platform and choose shifts.
+ The admin of the website can easily add new organizations, places and shifts. The software has a location based 
+ hierarchy (country / region / area / city) and has a hierarchy of organizations (organization / facility / workplace)
```
Comment (christophmeissner) [inline README.md]: > (organizations, facilities, tasks and workplaces)

**Comment by christophmeissner** [comment]
Thanks a lot!

File: README.md (hunk 2 above)
Comment (klinger) [inline README.md]: Good! I will try to reproduce this again, I think there was a problem with a repository not available.

**Comment by christophmeissner** [comment]
Thanks!

**Pull request merged**